component/Avatar/avatar.css

```diff
@@ -25,29 +26,29 @@
     text-align: center;
     padding-top: 4rem;
 }
-.mdm-text-avatar{
+.md-text-avatar{
     background-color:var(--COLOR-PRIMARY);
     font-size: 3.2rem;
     font-weight: 600;
     text-align: center;
     padding-top: 3rem;
     margin:2rem 2rem 0rem 2rem;
 }
-.small-text-avatar{
+.sm-text-avatar{
     background-color:var(--COLOR-PRIMARY);
     font-size: 2rem;
     font-weight: 600;
     text-align: center;
     padding-top: 1.2rem;
-    margin:4rem 2rem 0rem 0rem;
+    margin:4.5rem 2rem 0rem 0rem;
 }
-.xsmall-text-avatar{
+.xs-text-avatar{
     background-color:var(--COLOR-PRIMARY);
     font-size: 1.3rem;
     font-weight: 600;
     text-align: center;
     padding-top:6px;
-    margin:4.9rem 0rem 0rem 0rem;
+    margin:5.5rem 0rem 0rem 0rem;
```

Comment:
.md-text-avatar, .sm-text-avatar and .xs-text-avatar have a lot of common properties. you can combine them in one class.

Reply:
ok will look into it